### PR TITLE
Rootfull -> Rootful

### PR DESCRIPTION
**Pull request by ashley-cui**
This reverts commit cc3790f332d989440eb1720e24e3619fc97c74ee. And changes more docs. 

We can't change rootful to rootfull because `rootful` is written into the machine config. Changing this will break json unmarshalling, which will break existing machines.

[NO NEW TESTS NEEDED]

Signed-off-by: Ashley Cui <acui@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
We should PS about this.

**Comment by edsantiago** [comment]
Thank you @ashley-cui this is a particularly contentious peeve of mine. I appreciate your efforts. There is not, and never should be, any such thing as two-el "rootfull". It is an abomination.

File: vendor/github.com/containers/common/libnetwork/netavark/network.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type netavarkNetwork struct {
 	// networkRunDir is where temporary files are stored, i.e.the ipam db, aardvark config etc
 	networkRunDir string
 
-	// tells netavark whether this is rootless mode or rootfull, "true" or "false"
+	// tells netavark whether this is rootless mode or rootful, "true" or "false"
```
Comment (Luap99) [inline vendor/github.com/containers/common/libnetwork/netavark/network.go:30]: do not change vendor files

Comment (ashley-cui) [reply vendor/github.com/containers/common/libnetwork/netavark/network.go:30]: oops, did a vscode find and replace, will fix, thanks!

**Review by Luap99 — COMMENTED** [review]
LGTM

**Pull request merged**